libs/blocks/table/table.css

```diff
@@ -607,4 +607,8 @@ header.global-navigation {
     flex-direction: column;
     align-items: initial;
   }
+  
+  .table .action-area .con-button.button-l {
+    max-width: 70%;
```

Comment:
PR description states:
> Added min-width for table Cta button
which doesn't seem to match this declaration.

Furthermore, `70%` seemed a bit arbitrary to me, so I had a look at a few use-cases and I'm not sure this is the way to go. I limited the CTA's parent, `.action-area` to `max-width: 100%` and applied an `overflow: hidden` to the CTA. Here are a few use-cases comparing this approach (left column) to the one suggested in this PR (right column):

| 360px (70% still overflows) | 768px (70% spans multiple rows) | 1200px (70% spans multiple rows) |
| ------------- | ------------- | ------------- |
| ![Screenshot 2024-08-21 at 14 35 04](https://github.com/user-attachments/assets/fee9a34b-3f91-4d90-aa7f-aff0041cbdc8) | ![Screenshot 2024-08-21 at 14 36 30](https://github.com/user-attachments/assets/4c10306b-70e0-4605-a2a3-904ec87246d7) | ![Screenshot 2024-08-21 at 14 37 41](https://github.com/user-attachments/assets/ad5b06fb-b498-4ddd-ab40-b743800b8bd1) |